packages/inter-protocol/docs/governance.md

```diff
@@ -0,0 +1,113 @@
+# Inter Protocol Governance
+
+Inter Protocol is subject to two forms of governance. Like the entire chain, it's subject to BLDer DAO governance. This is sometimes called the "big hammer" because the DAO can approve running any code in the core ("core eval").
+
+There is also governance by a contract governor contract (see `@agoric/governance`). This allows invited members to execute governed API methods or set governed parameters. In the conventional configuration of the Inter Protocol contracts there is one governing group, which is called the "Economic Committee".
```

Comment:
```suggestion
There is also governance by a contract governor contract (see `@agoric/governance`). This allows elected members to execute governed API methods or set governed parameters. In the conventional configuration of the Inter Protocol contracts there is one governing group, which is called the "Economic Committee".
```

or just strike "invited"... or go with "committee members"

Reply:
wait... this is before we introduce the EC. "This allows a configurable electorate to execute..." would be more precise, though I'm not sure it's much more clear. hm.

"This delegates executing ... and setting ... to a body chosen when the contract is started"?